latest/index.bs

```diff
@@ -175,7 +175,294 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
                 └── n
 </pre>
 
+Tables {#table-layout}
+----------------------
+The following describes the expected layout for tabular data. OME-NGFF tables are compatible with the [AnnData model](https://github.com/theislab/anndata).
 
+<pre>
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images by image ID.
+│
+└── 123.zarr
+    └── table                 # The table group is a container which holds a table that is compatible with AnnData.
+        │                     # The table group MAY be in the root of the zarr file.
+        ├── .zgroup           # The table group MAY be in an image, labels, or points group.
+        |
+        ├── .zattrs           # `.zattrs` MUST contain "@type", which is set to `"ngff:region_table"`
+        |                     # `.zattrs` MUST contain "region", which is the path to the data the table is annotating.
+        |                     # "region" MUST be a single path (single region) or an array of paths (multiple regions).
+        |                     # "region" paths MUST be objects with a key "path" and the path value MUST be a string.
```

Comment:
@bogovicj , @joshmoore , does this track with your understanding of our proposal for paths?

Is the plan to add a section somewhere on paths that can be referenced?

Reply:
Current example at https://github.com/kevinyamauchi/ome-ngff-tables-prototype/blob/4fdde521b6b5514424f9c6508a8b1fc3a2cff86e/src/ngff_tables_prototype/writer.py#L60 generates a string e.g, "labels/label_image", not an object with "path".

Also, it'll be easier to validate etc if we have a list with a single entry for single `region` instead of a "single path or array of paths", since then we have to handle 2 different types of value.